Sources/Datadog/Core/Upload/RequestBuilder.swift

```diff
@@ -67,7 +67,7 @@ internal struct RequestBuilder {
 
             return HTTPHeader(
                 field: userAgentHeaderField,
-                value: .constant("\(sanitizedAppName)/\(appVersion) CFNetwork (\(device.model); \(device.osName)/\(device.osVersion))")
+                value: .constant("\(sanitizedAppName)/\(appVersion) CFNetwork (\(device.name); \(device.osName)/\(device.osVersion))")
```

Comment:
These kind of changes in this PR are OK. What used to work as `device.model` before, now is called `device.name` (`iPhone` | `iPad` | ...). The `device.model` is now reserved for exact info, e.g. `iPhone10,2`.